### DISCUSSION TITLE
Readonly Public View, Login to upload/edit

### DISCUSSION THREAD

**Discussion by arky** [filebrowser/filebrowser#2351] [Q&A]
When the user access the filebrowser, it will display all the public folders and allow users to preview and download them. However if you want to upload or modify a file in the public folder, you'll need to login as admin user. 

Wonder if such implementation is possible with filebrowser.

**Comment by RomainLG** [comment]
Hi, have you ever figured out a way of doing that?

**Reply by arky** [reply]
Nope, Still waiting for this feature.

**Reply by RomainLG** [reply]
I would love to have no authentification and disable all delete permission. I think this is feasible as is but not sure how...

**Reply by arky** [reply]
Perhaps someone from @filebrowser Dev team could provide some insight :crossed_fingers:

**Reply by RomainLG** [reply]
@o1egl
Hello,
Would you have advices on a setup with no authentication but no delete rights? Only upload.

**Comment by djmoran64** [comment]
I am interested in this as well.  open access for read only with the ability to login for more features.

**Comment by gletiecq** [comment]
Not on the dev team, but it's obvious that security is a you get it or you don't proposition in filebrowser.  And that's not a bad way to do it.  Seriously, what you're looking for could pose a massive problem for a lot of users out there that can't absolutely restrict access to fb as I assume (praying in fervent supplication to the LORD God Almighty that) you all can.  Because only HE is out to protect you from what you are about to make happen this way, and he might not answer these prayers.

For your purposes, you could have a "public user" with credentials "user" password "user" with read-only access to everything.  Then other users with write access would have different credentials.  May the people on your systems have mercy on your souls.
